site/src/components/state-info/WorkElectState.tsx

```diff
@@ -12,8 +12,7 @@ const WorkElectState = ({ info, jurisdictions }: Props) => (
       <h2>{ info.name }</h2>
       <div class="jurisdictions">
       {jurisdictions.map(({id, name}) => (
-            <a href={`https://workelections.com/j/${id}/${name.split(" ").join("-")}`}
-               target="_blank">{name}</a>
+            <a href={`/jurisdiction#${id}`} >{name}</a>
```

Comment:
link directly to our site

Reply:
there's probably a better way to handle URL params - happy to adapt but just did it with hash values for now

Reply:
Route param probably works best `/jurisdiction/:id`: https://github.com/ionic-team/stencil-router/wiki/Passing-data-to-routes

Reply:
Great - swapped to matching